pkg/handler/scale_jobs.go

```diff
@@ -7,20 +7,24 @@ import (
 	kedav1alpha1 "github.com/kedacore/keda/pkg/apis/keda/v1alpha1"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 func (h *ScaleHandler) scaleJobs(scaledObject *kedav1alpha1.ScaledObject, isActive bool, scaleTo int64, maxScale int64) {
-	// TODO: get current job count
-	h.logger.V(1).Info("Scaling Jobs")
+	runningJobCount := h.getRunningJobCount(scaledObject, maxScale)
+	h.logger.Info("Scaling Jobs", "Number of running Jobs ", runningJobCount)
+	h.logger.Info("Scaling Jobs")
 
 	if isActive {
 		h.logger.V(1).Info("At least one scaler is active")
 		now := metav1.Now()
 		scaledObject.Status.LastActiveTime = &now
 		h.updateScaledObjectStatus(scaledObject)
-		h.createJobs(scaledObject, scaleTo, maxScale)
+		h.createJobs(scaledObject, scaleTo, maxScale-runningJobCount)
```

Comment:
We should ensure maxScale - runningJobCount is not <= 0 here.

Reply:
That's an easy fix. Thanks!

Reply:
Done.